### PR TITLE
fix: package versioning

### PR DESCRIPTION
**Pull request by amritk**
This PR fixes some versioning while also adding a new strategy for testing packages locally.
```bash
sh ./pack-packages.sh
```
will create tarballs of the local package versions. This strategy is closer to an npm prod release than linking packages.

### CODE REVIEW

File: packages/api-client/package.json [hunk 1]
```diff
@@ -61,7 +61,6 @@
   "devDependencies": {
     "@scalar/api-client-proxy": "workspace:*",
     "@scalar/echo-server": "workspace:*",
-    "@scalar/use-codemirror": "^0.7.17",
```
Comment (hanspagel) [inline packages/api-client/package.json:64]: Why is that removed? 🤔 We’re importing from it in @scalar/api-client.

Comment (amritk) [reply packages/api-client/package.json:64]: it was duplicated, still in there

**Review by hanspagel — APPROVED** [review]
AWESOME, let’s try this

**Review by marclave — APPROVED** [review]
